src/control_flow/mod.rs

```diff
@@ -428,9 +475,7 @@ impl Visit for Analyzer<'_> {
       false
     };
 
-    if unreachable {
-      self.info.entry(n.span().lo).or_default().unreachable = true;
-    }
+    self.info.entry(n.span().lo).or_default().unreachable = unreachable;
```

Comment:
This will allocate an entry in hashmap even if it's reachable. Is this intended?

(I'm not talking about performance as it is a linter)

Reply:
Yes, it's intended. Without this change, there would be some statements that are not stored in the hashmap, which is not consistent with the comment appended to the `meta` method (the comment says that all statements can be queried).

https://github.com/denoland/deno_lint/blob/19bc382db7572d9b011f37eb90f1daa7df9279e0/src/control_flow/mod.rs#L26-L32